<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuteScoop</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQutescoop">
    <property name="title">
     <string>&amp;QuteScoop</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionDownload"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShowFriendsDock"/>
    <addaction name="actionShowSearch"/>
    <addaction name="actionShowMetars"/>
    <addaction name="actionClearAllFlightPaths"/>
    <addaction name="actionDisplayAllFlightPaths"/>
    <addaction name="actionHideAllWindows"/>
    <addaction name="separator"/>
    <addaction name="actionToggleFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionRememberPosition"/>
    <addaction name="actionDisplayAllFirs"/>
   </widget>
   <addaction name="menuQutescoop"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="searchDock">
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Search</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="buddy">
          <cstring>searchEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="searchResult">
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="metarDock">
   <property name="windowTitle">
    <string>METAR</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="buddy">
          <cstring>metarEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="metarEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnRefreshMetar">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="metarList">
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="metarDecoderDock">
   <property name="windowTitle">
    <string>METAR Decoder</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout">
     <property name="margin">
      <number>4</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="metarText">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'UnDotum'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="acceptRichText">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="friendsDock">
   <property name="windowTitle">
    <string>Friends</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout">
     <property name="margin">
      <number>4</number>
     </property>
     <item>
      <widget class="QTreeView" name="friendsList">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDownload">
   <property name="text">
    <string>&amp;Download Network Status</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Q&amp;T</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionShowSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Search Dock</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionToggleFullscreen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Toggle Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F11</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionClearAllFlightPaths">
   <property name="text">
    <string>&amp;Hide All Flight Routes</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionDisplayAllFlightPaths">
   <property name="text">
    <string>Show &amp;All Flight Routes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F6</string>
   </property>
  </action>
  <action name="actionShowMetars">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Metar Dock</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionRememberPosition">
   <property name="text">
    <string>&amp;Remember Map Position</string>
   </property>
  </action>
  <action name="actionHideAllWindows">
   <property name="text">
    <string>Hide all &amp;Windows</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionShowFriendsDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Friends Dock</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionDisplayAllFirs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display All Sector &amp;Boundaries</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShowSearch</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>searchDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowMetars</sender>
   <signal>toggled(bool)</signal>
   <receiver>metarDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowFriendsDock</sender>
   <signal>toggled(bool)</signal>
   <receiver>friendsDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metarDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShowMetars</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>friendsDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShowFriendsDock</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
